admin/sql/updates/20210507-mbs-11652-artist-series.sql

```diff
@@ -0,0 +1,54 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+\set ARTIST_PART_OF_SERIES_GID 'generate_uuid_v3(''6ba7b8119dad11d180b400c04fd430c8'', ''http://musicbrainz.org/linktype/artist/series/part_of'')'
+\set LINK_ATTRIBUTE_TYPE_NUMBER_ID '788'
+
+-----------------------
+-- CREATE NEW VIEWS  --
+-----------------------
+
+CREATE OR REPLACE VIEW artist_series AS
+    SELECT entity0 AS artist,
+           entity1 AS series,
+           las.id AS relationship,
+           link_order,
+           las.link,
+           COALESCE(text_value, '') AS text_value
+    FROM l_artist_series las
+    JOIN series s ON s.id = las.entity1
+    JOIN link l ON l.id = las.link
+    JOIN link_type lt ON (lt.id = l.link_type AND lt.gid = :ARTIST_PART_OF_SERIES_GID)
+    LEFT OUTER JOIN link_attribute_text_value latv ON (latv.attribute_type = :LINK_ATTRIBUTE_TYPE_NUMBER_ID AND latv.link = l.id)
+    ORDER BY series, link_order;
+
+-------------------------
+-- INSERT INITIAL DATA --
+-------------------------
+
+-- Part-of-series rel
+
+INSERT INTO link_type (gid, entity_type0, entity_type1, entity0_cardinality,
+                       entity1_cardinality, name, description, link_phrase,
+                       reverse_link_phrase, long_link_phrase) VALUES
+    (
+        :ARTIST_PART_OF_SERIES_GID,
+        'artist', 'series', 0, 0, 'part of',
+        'Indicates that the artist is part of a series.',
+        'part of', 'has parts', 'is a part of'
+    )
+    RETURNING id, gid, entity_type0, entity_type1, name, long_link_phrase;
```

Comment:
This might need adding the number attribute connection too, after inserting the link_type? Or am I missing something :)

Reply:
Added at https://github.com/metabrainz/musicbrainz-server/pull/2098/commits/3cfc07e50d520b2e0b2d1a839e2f87049763a2f8#diff-c0b8567c3b5ab5956ce6cf3d7eef6db23261fe7d4558fab77e027efc3ecd3ab5R41-R47